source/features/recently-pushed-branches-enhancements.tsx

```diff
@@ -7,26 +7,26 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 import {getRepo} from '../github-helpers';
 
-const fragmentURL = `/${getRepo()!.nameWithOwner}/show_partial?partial=tree%2Frecently_touched_branches_list`;
-const selector = `[data-url='${fragmentURL}' i], [src='${fragmentURL}' i]`;
+const fragmentURL = (): string => `/${getRepo()!.nameWithOwner}/show_partial?partial=tree%2Frecently_touched_branches_list`;
+const selector = (): string => `[data-url='${fragmentURL()}' i], [src='${fragmentURL()}' i]`;
```

Comment:
Should I put this in its own PR? its breaking all non repo features. Or Do you think we will merge by Thursday?

Reply:
Yeah you/I can merge it by Wednesday

Reply:
But actually, if you have time, there are 2 possible improvements:

- if only one of `data-src` and `src` are necessary, drop the other one
- merge the 2 init functions and avoid these globals altogether. You can replicate the `onetime` part with a plain global boolean.

Reply:
Will git it a shot

Reply:
> * if only one of `data-src` and `src` are necessary, drop the other one

f40da70ed947dfdde79475bdee9b93f6848e9202

I am not sure what you were fixing here. Remeber?

Reply:
The selector change was not introduced by that commit, but the commit did separate the comment from the code it commented. 

That comment explains why there are 2 selectors; it should be reinstated and probably the selector won’t have to be changed.